docker/wikidata-bot.service

```diff
@@ -1,4 +1,3 @@
 #!/bin/bash
 
-cd /code
-exec /code/run.py
+exec run-consul-template -config /etc/consul-template.conf
```

Comment:
like in other services, we should name this file more explicitly so that we know the service that it refers to: `consul-template-mb2wdbot.conf`